server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -188,4 +188,7 @@ grant {
   permission java.io.FilePermission "/sys/fs/cgroup/memory", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/memory/-", "read";
 
+  permission java.net.NetPermission "getProxySelector";
```

Comment:
Most of these permissions are already defined (please check the policy) but why do you need to modify them? It should come with tracing plugin

Reply:
> This was added to enable OTLPGrpacSpanExporter for testing. Will add later if needed.

This is a valid case (since we would need to support different exporters) but those permissions should go to otel plugin (not core)

Reply:
This permission is not working somehow from the plugin security file. I need to debug that. But here this is the accidental change and I am removing this.